server/index.js

```diff
@@ -27,7 +27,13 @@ app.use(express.static('./build'));
 app.use(express.static('./doc'));
 app.use(cors());
 app.use(logger('dev'));
-app.use(express.json());
+app.use((req, res, next) => {
+  if (req.originalUrl === '/webhook') {
```

Comment:
What does this do?

Reply:
This makes it so the webhook endpoint receives raw instead of parsed data. The stripe verification to make sure the request hitting our webhook came from stripe requires raw data instead of json (I have no clue why...). The rest of the endpoints remain unchanged and will be parsed since they are not /webhook.